### DISCUSSION TITLE
重新安装后收不到恢复码

### DISCUSSION THREAD

**Discussion by yangyvan** [getlantern/lantern-forum-cn#341] [Q&A] [answered]
#70937029 不解决问题

yangyvan@hotmail.com 购买专业版邮箱 请发我恢复码

**Comment by MaxMKMax** [comment, marked as answer]
@yangyvan 给您带来的不便，敬请谅解。今晚我将帮助您请求团队向您发送恢复代码。请明天查看您的电子邮件。谢谢。

**Reply by 888sssjjjjgfyh** [reply]
wangxiaoguang1690@gmail.com。同样问题，没了专业版。而且新版本的7.8.4同样不行。
